fontforge/splineutil.c

```diff
@@ -7953,3 +7953,21 @@ bigreal DistanceBetweenPoints( BasePoint *p1, BasePoint *p2 )
     t = sqrt( t );
     return t;
 }
+
```

Comment:
Don't need this anymore.

Reply:
Yeah, you're right. My excuse for keeping it in is pretty thin, it's "it's kind of nice to know how many points off we are if we're working on making a glyph compatible."

If you're convinced by that, I'll remove this tomorrow

Reply:
If you really want to keep it you can but you'll have to correct the Cubic path (it should only count on-curve points) and verify that `SSTtfNumberPoints()` really does what it needs to. Given that the function is unnecessary the work also seems unnecessary.

Reply:
I guess an alternative is, in InterpolationSanity, if we hit the `(bnonextpt != ononextpt)` branch, just `int oextras = 0; while (osp != NULL) { extras++; osp = osp->next->to; }` if `bnonextcp`, then similar if `ononextcp`, then we can report in the warning something like &laquo;(base/other has %d extra unexpected points)&raquo;.

That's what I'll end up doing tomorrow. If I get some sleep :P

Reply:
Or just have a quick internal two-line loop to increment looking for `first` or `NULL`